config/experiment/simple_a2c/cartpole.yaml

```diff
@@ -2,5 +2,5 @@
 defaults:
   - override /services/actor:
       - simple_a2c
-  - override /run: simple_a2c
+  - override /run: headless_play
```

Comment:
Why?

Reply:
It is just leftover from the previous version of the next branch. I have changed ` override /run: headless_play` to `override /run: simple_a2c`